examples/rhfe_single.py

```diff
@@ -0,0 +1,178 @@
+# This script computes the relative hydration free energy with a single topology protocol.
+# It checks to make sure the estimated dG is in agreement between the following three methods:
+# 1) Two seperate absolute free energy calculations
+# 2) Relative free energy with full atom-mapping
+# 3) Relative free energy with partial atom-mapping (4D-decoupling)
+
+import os
+import argparse
+import numpy as np
+
+from rdkit import Chem
+from rdkit.Chem import rdFMCS
+from rdkit.Chem import AllChem
+
+from fe import topology
+from md import builders
+from md import minimizer
+
+import functools
+
+from ff import Forcefield
+from ff.handlers.deserialize import deserialize_handlers
+
+import multiprocessing
+
+from fe import free_energy
+
+
+def wrap_method(args, fn):
+    gpu_idx = args[0]
+    os.environ['CUDA_VISIBLE_DEVICES'] = str(gpu_idx)
+    return fn(*args[1:])
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(
+        description="Relative Hydration Free Energy Consistency Testing",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--num_gpus",
+        type=int,
+        help="number of gpus",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_vacuum_windows",
+        type=int,
+        help="number of vacuum lambda windows",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_solvent_windows",
+        type=int,
+        help="number of solvent lambda windows",
+        required=True
+    )
+
+    parser.add_argument(
+        "--num_equil_steps",
+        type=int,
+        help="number of equilibration lambda windows",
```

Comment:
in the next couple arguments, should "lambda windows" be "MD steps per window" or similar?

Reply:
Will fix